### PR TITLE
[replication] Added upgrade page and updated install info

### PR DESCRIPTION
**Pull request by walker2**
# Description
- Added upgrade page with info on about how to upgrade replication controller using `repctl` or `helm` 
- Added note about repctl install logs 
- Updated installation instructions when using script

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/66 |

Related PRs: 

| GitHub Pull Request# |
| -------------- |
| https://github.com/dell/csm-replication/pull/13 |
| https://github.com/dell/csm-replication/pull/14 |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [x] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

**Review by IliaRN — APPROVED** [review]

File: content/docs/replication/deployment/install-repctl.md [hunk 1]
```diff
@@ -7,8 +7,7 @@ description: Installation of CSM for Replication using repctl
 
 ## Install Replication Walkthrough
 
-Here are simple steps on how you can start using Container Storage Modules (CSM) for Replication with
-help from `repctl`
```
Comment (rsedlock1958) [inline content/docs/replication/deployment/install-repctl.md:11]: Change line 10 to read: You can start using Container Storage Modules (CSM) for Replication with help from repctlusing these simple steps:

File: content/docs/replication/deployment/install-repctl.md [hunk 2]
```diff
@@ -43,3 +42,6 @@ help from `repctl`
       ```shell
       ./repctl create pvc --rg <rg-name> -t <target-namespace> --dry-run=false
       ```
+
+
+> Note: all `repctl` output is saved alongside with `repctl` binary in `repctl.log` file and can be attached to any installation troubleshooting requests
```
Comment (rsedlock1958) [inline content/docs/replication/deployment/install-repctl.md:47]: Ad the word "the" before "repctl.log"

**Review by maxan98 — DISMISSED** [review]

**Comment by mjsdell** [comment]
@walker2 This needs to go into the release1.1 branch. cc: @shanmydell

**Review by maxan98 — DISMISSED** [review]

File: content/docs/replication/upgrade.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+1. Update the `image` value in the values files to reference the new CSM Replication sidecar image or use a new version of the csm-replication helm chart
+2. Run the install script with the option `--upgrade` by running: `cd ./scripts && ./install.sh --values ./myvalues.yaml --upgrade`
+3. Run the same command on the second Kubernetes cluster if you use multi-cluster replication topology
+
+> Note: Upgrade won't override currently existing ConfigMap, even if you change templated values in myvalues.yaml file. If you want to change the logLevel - edit ConfigMap from within the cluster using `kubectl edit cm -n dell-replication-controller dell-replication-controller-config`
+
+
+### Upgrading with repctl
+
+> Note: These steps assume that you already have `repctl` configured to use correct clusters, if you don't know how to do that please refer to [installing with repctl](./deployment/install-repctl.md)
```
Comment (shanmydell) [inline content/docs/replication/upgrade.md:45]: This link will not work with Hugo , please modify the same

**Review by shanmydell — CHANGES_REQUESTED** [review]

File: content/docs/replication/upgrade.md (hunk 3 above)
Comment (walker2) [reply content/docs/replication/upgrade.md]: Changed to a link that works with the latest commit

**Review by shanmydell — APPROVED** [review]

**Review by maxan98 — APPROVED** [review]

**Pull request merged**